src/VizbeeCastButtonPressable/index.js

```diff
@@ -0,0 +1,49 @@
+import React, { useRef } from "react";
+import {
+  NativeModules,
+  findNodeHandle,
+  TouchableWithoutFeedback,
+  Platform,
+  UIManager,
+} from "react-native";
+import { VizbeeCastButton } from "../../";
+
+const VizbeeCastButtonViewManager = NativeModules.VizbeeCastButtonView || {};
+
+const VizbeeCastButtonPressable = ({ style }) => {
+  const buttonRef = useRef();
+
+  const handlePress = () => {
+    // Get the view tag using findNodeHandle
```

Comment:
Why can't it simply be `VizbeeManager.smartprompt()`?

Reply:
It is in iOS but not in Android as in andriod we extend a SimpleViewManager and not a Module like we did for VizbeeNativeManager. Also we need to know the view to execute the performClick Method.

Reply:
It's there in Android too `smartCast()`

Reply:
The question is, why does it have to attach to view's performClick()? when smartCast() does the same functionality.